src/poliastro/twobody/orbit.py

```diff
@@ -839,6 +852,53 @@ def frozen(
                 )
             )
 
+    def calculate_pass(self, lat, long, H):
+        a = self.a  # semimajor axis
+        e = self.ecc  # eccentricity
+        i = self.inc  # equatorial inclination
+        O = self.long_asc  # longitude of ascending node
+        o = self.argp  # argument of perigee
+        T = self.period  # time of perifocal passage
+
+        n = self.n  # mean motion
+        ar = self.attractor.r  # equatorial radius
+        f = 0  # TODO: flattening
+
+        t0 = 1  # TODO: epoch universal time (rad)
+        th0 = 1  # TODO: sidereal station time (rad)
```

Comment:
Having selected the reference time, according to the Vallado equation (3-43), this will be equal the longitude plus `erfa.gmst06`:

```
In [20]: t = Time.now()                                                                                                                 

In [21]: t.ut1.jd1                                                                                                                      
Out[21]: 2458701.0

In [22]: t.ut1.jd2                                                                                                                      
Out[22]: 0.432900555821122

In [23]: t.tt.jd1                                                                                                                       
Out[23]: 2458701.0

In [24]: t.tt.jd2                                                                                                                       
Out[24]: 0.43370315784722224

In [25]: erfa.gmst06(t.ut1.jd1, t.ut1.jd2, t.tt.jd1, t.tt.jd2)                                                                          
Out[25]: 5.061874805765537
```

Reply:
A comment about the applicability of this for other planets. From `gmst06` docs:

> 2) Both UT1 and TT are required, UT1 to predict the Earth rotation
   and TT to predict the effects of precession.  If UT1 is used for
   both purposes, errors of order 100 microarcseconds result.

Source code:

https://github.com/liberfa/erfa/blob/95bdbb87dc0ac444c35da95cbaa1b46b9500c6bb/src/gmst06.c#L68-L79

After implementing the Earth case, we should carefully reflect on how to apply the formulas of WGCRE here, and probably remove the use of ERFA completely. But let's leave that for a future PR.

Reply:
Is ``erfa`` an astropy module? I can't seem to find it.

Reply:
Oops! Yes, it's `from astropy import _erfa as erfa`

Reply:
@Juanlu001 Just another question regarding your comment: 

> this will be equal to the longitude plus erfa.gmst06

the output of ``erfa.gmst06`` is a float while longitude is calculated in rads. Do we need to take the value of the longitude or can we also convert ``erfa.gmst06`` to rads (presumably, theta must be in rads anyway)

Reply:
According to the docs:

https://github.com/liberfa/erfa/blob/7009153a0fdbc2783e9a3442ddde88b9d9e8b5ef/src/gmst06.c#L15-L16

`erfa.gmst06` is already in radians, so at most we would need to attach a unit.